src/views/landing/index.js

```diff
@@ -184,10 +184,12 @@ const Landing = ({
 Landing.init = ({
   services: {ui: {toggleModal}}
 }) => ({
-  toggleSignUpModal: () => {
+  toggleSignUpModal: (e) => {
+    let modalName = 'signupver1'
+    if (e.target.className.indexOf('version2') > 0){ modalName = 'signupver2' }
```

Comment:
```js
const version = e.target.className.split(' ').find(item => /version/.test(item))
const modalName = {
  version1: 'signupver1',
  version2: 'signupver2',
  version3: 'signupver3'
}[version || 'version1']
```

Or even better, if you change `signupver1` (and other names) to `signup-version1` you can do this:

```js
const version= e.target.className.split(' ').find(item => /version/.test(item))
const modalName = `signup-${version || 'version1'}`
```